os-server/src/main/java/app/openschool/course/api/dto/UserEnrolledCourseDto.java

```diff
@@ -2,7 +2,7 @@
 
 import java.time.LocalDate;
 
-public class UserCourseDto {
+public class UserEnrolledCourseDto {
```

Comment:
why this is renamed?

Reply:
To be different from dto of mentor's courses.